<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QualityControlDialog</class>
 <widget class="QTabWidget" name="QualityControlDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>799</width>
    <height>379</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TabWidget</string>
  </property>
  <property name="currentIndex">
   <number>0</number>
  </property>
  <widget class="QWidget" name="tab">
   <attribute name="title">
    <string>质控登记</string>
   </attribute>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>9</y>
      <width>781</width>
      <height>321</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QListView" name="ItemListView">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="sizeIncrement">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">alternate-background-color: rgb(255, 170, 0);
color: rgb(0, 0, 255);
font: 63 11pt &quot;Ubuntu&quot;;</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <layout class="QFormLayout" name="formLayout">
           <item row="0" column="0">
            <widget class="QLabel" name="label">
             <property name="styleSheet">
              <string notr="true">color: rgb(0, 85, 255);
font: 11pt &quot;Vemana2000&quot;;</string>
             </property>
             <property name="text">
              <string>质控值</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="edt_QcValue"/>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_2">
             <property name="styleSheet">
              <string notr="true">color: rgb(0, 85, 255);
font: 11pt &quot;Vemana2000&quot;;</string>
             </property>
             <property name="text">
              <string>质控日期</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QDateTimeEdit" name="dt_QcDate">
             <property name="buttonSymbols">
              <enum>QAbstractSpinBox::UpDownArrows</enum>
             </property>
             <property name="dateTime">
              <datetime>
               <hour>0</hour>
               <minute>0</minute>
               <second>0</second>
               <year>2000</year>
               <month>3</month>
               <day>1</day>
              </datetime>
             </property>
             <property name="displayFormat">
              <string>yyyy-MM-dd hh:mm</string>
             </property>
             <property name="calendarPopup">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QPushButton" name="btn_QC_Saves">
             <property name="font">
              <font>
               <family>WenQuanYi Zen Hei</family>
               <pointsize>12</pointsize>
               <weight>50</weight>
               <bold>false</bold>
              </font>
             </property>
             <property name="cursor">
              <cursorShape>PointingHandCursor</cursorShape>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color: rgb(85, 170, 255);
color: rgb(0, 0, 0);</string>
             </property>
             <property name="text">
              <string>保存</string>
             </property>
             <property name="icon">
              <iconset resource="rc.qrc">
               <normaloff>:/image/Save.png</normaloff>:/image/Save.png</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>32</width>
               <height>32</height>
              </size>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QFormLayout" name="formLayout_2">
           <item row="1" column="0">
            <widget class="QLabel" name="label_3">
             <property name="styleSheet">
              <string notr="true">color: rgb(0, 85, 255);
font: 11pt &quot;Vemana2000&quot;;</string>
             </property>
             <property name="text">
              <string>靶值</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLineEdit" name="edt_TargetValue"/>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_4">
             <property name="styleSheet">
              <string notr="true">color: rgb(0, 85, 255);
font: 11pt &quot;Vemana2000&quot;;</string>
             </property>
             <property name="text">
              <string>CV值</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QLineEdit" name="edt_CvValue"/>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="label_5">
             <property name="styleSheet">
              <string notr="true">color: rgb(0, 85, 255);
font: 11pt &quot;Vemana2000&quot;;</string>
             </property>
             <property name="text">
              <string>样本批号</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QPushButton" name="btn_Calculate">
             <property name="font">
              <font>
               <family>WenQuanYi Zen Hei</family>
               <pointsize>12</pointsize>
               <weight>50</weight>
               <bold>false</bold>
              </font>
             </property>
             <property name="cursor">
              <cursorShape>PointingHandCursor</cursorShape>
             </property>
             <property name="autoFillBackground">
              <bool>false</bool>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color: rgb(85, 170, 255);
color: rgb(0, 0, 0);</string>
             </property>
             <property name="text">
              <string>计算</string>
             </property>
             <property name="icon">
              <iconset resource="rc.qrc">
               <normaloff>:/image/Compute.png</normaloff>:/image/Compute.png</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>32</width>
               <height>32</height>
              </size>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="edt_BatchNo"/>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTableView" name="tb_SubItemView">
         <property name="styleSheet">
          <string notr="true">alternate-background-color: rgb(255, 170, 0);
color: rgb(0, 0, 255);
font: 63 11pt &quot;Ubuntu&quot;;</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="rc.qrc"/>
 </resources>
 <connections/>
</ui>
